src/plugins/size/README.md

```diff
@@ -0,0 +1,41 @@
+- infos = Information about the size plugin is in keys below
+- infos/author = Thomas Waser <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = conv
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained conformant compatible coverage specific unittest shelltest tested nodep libc configurable experimental
+- infos/metadata = transform/size
+- infos/description = transforms WIDTHxHEIGHT values into separate keys
+
+## Introduction
+
+The `size` plugin transforms values with the format `WIDTH<separator>HEIGHT` into a separate `Width` and `Height` key. The metakey `transform/size` triggers the conversion.
+
+## Configuration
+
+By default `x` is assumed as separator. If the plugin configuration `/separator` is set, its value is used instead. If set, `transform/size`'s value takes precedence.
```

Comment:
the `transform/size` value (per key) overrides `/separator` (plugin configuration), neither is needed and it defaults to `x` (hardcoded).
LCDproc only uses `x`, but it could be useful for similar things like aspect ratios which are usually separated by a `:`
but we can drop it if there's no need for it